readthedocs/source/installation.rst

```diff
@@ -70,6 +70,19 @@ An installation with ``pip`` should work as well. See the `CAMB <https://github.
 repo for more details. Note that if you installed CCL with ``conda``, ``camb``
 should already be in your environment.
 
+.. _getting-fast-pt:
+
+Getting FastPT
+==============
+
+To use ``FastPT`` with CCL, you can install it with:
```

Comment:
and here.

Reply:
Done. I think you were also looking at an old version without the hyphens.